assets/global.js

```diff
@@ -764,9 +784,17 @@ class SlideshowComponent extends SliderComponent {
     } else if (isLastSlide && event.currentTarget.name === 'next') {
       this.slideScrollPosition = 0;
     }
-    this.slider.scrollTo({
-      left: this.slideScrollPosition,
-    });
+    if (this.querySelector('.announcement-bar-slider')) {
+      setTimeout (() => {
+        this.slider.scrollTo({
+          left: this.slideScrollPosition,
+        });
+      }, 200);
```

Comment:
I'd suggest saving the `200` value as a constant, because I see it's being used in a few places